doc/sphinxext/numpy_ext/docscrape.py

```diff
@@ -435,7 +436,11 @@ def __init__(self, func, role='func', doc=None, config={}):
             func, func_name = self.get_func()
             try:
                 # try to read signature
-                argspec = inspect.getargspec(func)
+                if 3 > sys.version_info[0]:
+                   argspec = inspect.getargspec(func)
+                else:
+                   # getargspec deprecated in python3
+                   argspec = inspect.signature(func)
```

Comment:
@amueller See comments in [joblib](https://github.com/joblib/joblib/pull/375/).  It may not be appropriate to change this file.  What do you think?